### PR TITLE
Fix balance pagination

### PR DESCRIPTION
**Pull request by dissident**
https://trello.com/c/Fxrlz8AM/296-balance

### CODE REVIEW

File: app/assets/javascripts/statistics.js [hunk 1]
```diff
@@ -64,6 +71,18 @@ var getScale = function getScale() {
   return scale
 }
 
+var prohibitStep = function prohibitStep(elementSelector) {
```
Comment (shir) [inline app/assets/javascripts/statistics.js]: @dissident if you assign function to variable you don't need to name it. So:
```js
var prohibitStep = function (..) { ... }
```
or
```js
function prohibitStep(...) {...}
```

Comment (shir) [inline app/assets/javascripts/statistics.js]: @dissident I think you may write it simpler using `toggleClass`:
```js
function toggleStep(elementSelector, state) {
  element = $(elementSelector);
  element.find('a').toggleClass('hide', state);
  element.find('span').toggleClass('hide', state);
}
```
And then use
```js
toggleStep('.right_step_wrapper', step == 0)
```

Comment (dissident) [inline app/assets/javascripts/statistics.js]: @shir it's not working correctly for my case. One state can be false or true, and I need add class hide to one element and delete from another, but the state in this toggleClass work another way, he adds the only class if true, and remove only if false.

Comment (shir) [inline app/assets/javascripts/statistics.js]: @dissident 
```js
element.find('a').toggleClass('hide', state);
element.find('span').toggleClass('hide', !state);
```

File: app/assets/javascripts/statistics.js [hunk 2]
```diff
@@ -64,6 +71,12 @@ var getScale = function getScale() {
   return scale
 }
 
+function toggleStep(elementSelector, state) {
+  element = $(elementSelector);
```
Comment (shir) [inline app/assets/javascripts/statistics.js]: @dissident didn't notice it before. You miss `var` here too.

**Pull request merged**